package/gluon-web-model/luasrc/usr/lib/lua/gluon/web/model/classes.lua

```diff
@@ -97,7 +97,7 @@ end
 function Node:resolve_depends()
 	local updated = false
 	for _, node in ipairs(self.children) do
-		update = updated or node:resolve_depends()
+		updated = updated or node:resolve_depends()
```

Comment:
This one is an actual bugfix that changes logic, let's move that to a separate commit (can still be the same PR).

Reply:
It is move to a separate commit